lockfile/audit/src/index.ts

```diff
@@ -1,10 +1,13 @@
+import path from 'path'
 import { PnpmError } from '@pnpm/error'
 import { AgentOptions, fetchWithAgent, RetryTimeoutOptions } from '@pnpm/fetch'
 import { GetAuthHeader } from '@pnpm/fetching-types'
 import { Lockfile } from '@pnpm/lockfile-types'
 import { DependenciesField } from '@pnpm/types'
 import { lockfileToAuditTree } from './lockfileToAuditTree'
 import { AuditReport } from './types'
+import { searchPackages } from '@pnpm/list'
+import { PackageNode } from '@pnpm/reviewing.dependencies-hierarchy'
```

Comment:
Let's reexport this type from `@pnpm/list` to avoid installing an additional dependency just for this.